xbmc/utils/Variant.cpp

```diff
@@ -682,6 +682,17 @@ bool CVariant::operator==(const CVariant &rhs) const
   return false;
 }
 
+void CVariant::reserve(unsigned int length)
```

Comment:
Why don't you use `size_t` instead of `unsigned int` since `std::vector::reserve() usually expects `size_t`?